dt-assets/js/shared-functions.js

```diff
@@ -186,6 +186,15 @@ window.API = {
       user_id: userId,
     }),
 
+  getTimeMetricsByYear: (postType, field, year) =>
+    makeRequest('GET', `metrics/time_metrics_by_year/${postType}/${field}`),
+
+  getTimeMetricsByMonth: (postType, field, year) =>
+    makeRequest('GET', `metrics/time_metrics_by_month/${postType}/${field}/${year}`),
+
+  getFieldSettings: (postType) =>
+    makeRequest('GET', `metrics/field_settings/${postType}`),
+
```

Comment:
These are specific to the metrics section. So let's keep them in the time-charts.js

Reply:
resolved 7afd3930

Reply:
what i meant for here is to not include the functions for metric enpoints in the shared-function.js since the endpoints might not be commonly available
We could either move these code declarations to time-charts.js, or call the makeRequest function from the time-charts.js

Reply:
ah :slightly_smiling_face:  gotcha! sure thing

Reply:
resolved 58b10bd5